pyproject.toml

```diff
@@ -25,8 +25,9 @@ dependencies = [
     "requests >=2.28.0,<3.0.0",
     "pydriller >=2.0,<3.0.0",
     "yamale >=4.0.3,<5.0.0",
+    "jinja2 >=3.1.2,<4.0.0",
     "packaging >=21.3,<24.0.0",
-    "jinja2 >=3.1.2,<4.0.0"
+    "SQLAlchemy==1.4.46"
```

Comment:
It looks like this will at least require rewriting all the orm-mapped classes to use `Mapped` and `mapped_column` instead of `Column`, the migration pathway is documented here https://docs.sqlalchemy.org/en/14/changelog/migration_20.html. We don't have too many tables so it should hopefully be relatively straightforward though

Reply:
Let's go for it then :+1: